version.php

```diff
@@ -5,14 +5,14 @@
  *
  * @package    tinymce_recordrtc
  * @author     Jesus Federico  (jesus [at] blindsidenetworks [dt] com)
- * @copyright  2016 Blindside Networks Inc.
+ * @copyright  2017 Blindside Networks Inc.
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2016120102;            // The current plugin version (Date: YYYYMMDDXX).
+$plugin->version   = 2017051900;            // The current plugin version (Date: YYYYMMDDXX).
```

Comment:
We don't change the versions like this in the Moodle context. As this is an enhancement to the original version, just increase the last two digits.

$plugin->version   = 2016120103;